openreview/venue/webfield/seniorAreaChairsWebfield.js

```diff
@@ -0,0 +1,30 @@
+// Webfield component
+return {
+    component: 'SeniorAreaChairConsole',
```

Comment:
I think this and the PC console template should be changed to use 2 spaces for indentation like all the other JS files.

Reply:
I will add this to another PR